src/css/components/modal.less

```diff
@@ -109,3 +119,99 @@
     }
   }
 }
+
+.modal.dialog {
+  color: @modal-dialog-text-color;
+  font-size: @modal-font-size;
+
+  .modal-header {
```

Comment:
If we're nesting there's no need for repeating the "namespace", imho. 
`.modal.dialog.header`, `.modal.dialog.content` etc should be enough.